src/questions.ts

```diff
@@ -0,0 +1,40 @@
+import * as inquirer from 'inquirer';
+import { basename } from 'path';
+
+const packageQuestions: inquirer.Question[] = [{
+	type: 'input',
+	name: 'package',
+	message: 'What Package to do you want to theme?'
+}];
+
+function getFileQuestions(packageName: string, files: string[], cssDataFileExtension: string): inquirer.Question[] {
```

Comment:
you can export these one by one, ie `export function getFileQuestions()...`

Reply:
Just wondering, do you think that's better? I kind of like 

```ts
export {
	packageQuestions,
	getFileQuestions,
	askForPackageNames,
	askForDesiredFiles
}
```

Because it's easy to scan for exports? But it's not a big thing, we can change it to `export function getFileQuestions` if desired 👍